web/themes/joinup/templates/form/container--edit-actions--search-bar.html.twig

```diff
@@ -0,0 +1,35 @@
+{#
+/**
+ * @file
+ * Theme override of a container used to wrap child elements.
+ *
+ * Used for grouped form items. Can also be used as a theme wrapper for any
+ * renderable element, to surround it with a <div> and HTML attributes.
+ * See \Drupal\Core\Render\Element\RenderElement for more
+ * information on the #theme_wrappers render array property, and
+ * \Drupal\Core\Render\Element\container for usage of the container render
+ * element.
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the containing element.
+ * - children: The rendered child elements of the container.
+ * - has_parent: A flag to indicate that the container has one or more parent
+     containers.
+ *
+ * @see template_preprocess_container()
+ */
+#}
+{%
+  set classes = [
+    has_parent ? 'js-form-wrapper',
+    has_parent ? 'form-wrapper',
+    is_collection ? 'mdl-grid',
+    is_collection ? 'mdl-grid--no-spacing',
+    is_collection ? 'vertical-tabs__actions',
+    'form__actions-wrapper',
+  ]
+%}
+
+
+    {{ children }}
```

Comment:
This modification needs some attention:
- classes are set but never used
- only children is shown, where-as the correction most probably is possible without stripping the rest

Please try to approach the problem with CSS rather than strip-down template.